verification/cbmc/proofs/run-cbmc-proofs.py

```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/env python3
```

Comment:
Why do we need to include this as a file, rather than having it in the submodule and just putting a symlink to that?

Reply:
This is a decision we made in the starter kit. The reason is that the run script differs subtly between projects, so having a symlink is no good.

As a concrete example, as soon as we switch off batch on E-SDK, we shall change the proof marker file from `cbmc-batch.yaml` to `cbmc-proof.txt`, since the former file is no longer meaningful. The run script will therefore have to change to look for the latter file.

Reply:
Ahh, makes sense. It seems like once we're standardized, we should maybe move to the symlink model.